Popover: ignore onClose() if focus is on opener (#472)
aragon/aragon-ui: src/components/Popover/README.md

Reference to the "opener"which is what the Popover is positioning relative to.

### `closeOnOpenerFocus`

| Type      | Default value |
| --------- | ------------- |
| `Boolean` | `false`       |

> Do not immediatly reopen the Popover when the opener gets focused while being opened, making it possible to toggle its visibility from the opener, if its placement allows it.

Author: ```suggestion
While the Popover is opened, do not immediately close the Popover when the opener gets focused. This makes it possible to toggle its visibility from the opener, if its placement allows it.
```
